README.md

```diff
@@ -47,6 +47,48 @@ brew install instructure-bridge/truss-cli/truss-cli
 go get github.com/instructure-bridge/truss-cli truss
 ```
 
+## Usage
+
+### Secrets
+
+The `truss secrets` command makes it easier to manage secrets in git, and
+synchronize them across multiple Truss Vault servers. Start by creating a
+`secrets.yaml` file.
+
+```yaml
+# secrets.yaml
+transit-key-name: my-project
+environments:
+  edge-cmh: # As declared in ~/.truss.yaml
+    secret: ./secrets/edge-cmh # Relative to `pwd`
```

Comment:
I would expect this to be the vault secret path, which would have nothing to do with pwd

Reply:
Yeah, the naming between "secret" and "path" is weird... One is supposed to be where the local file is stored, the other is where they're stored in Vault. Any suggestions for naming here?